### DISCUSSION TITLE
Inability to view a  website hosted on github live

### DISCUSSION THREAD

**Discussion by Encoded99** [community/community#55654] [New to GitHub]
I am trying to view  a code i hosted on git hube live , its actually a website, i read the docs and i was told to got to settings and then click on page and then click on live,  but i cant see the option for visit site

**Comment by queenofcorgis** [comment]
Hi @Encoded99 , we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.

**Comment by Encoded99** [comment]
The issue has been resolved, thanks
